commons/src/main/java/com/skytouch/entity/ProductEntity.java

```diff
@@ -8,10 +8,12 @@
 import javax.persistence.Column;
 import javax.persistence.Entity;
 import javax.persistence.Id;
+import javax.persistence.Table;
 
-@Entity
 @Getter
 @Setter
+@Entity
+@Table(name = "products")
```

Comment:
You shouldn't be using `@Entity`, `@Table`, `@Column`, `@Id` annotations as you are using stored procedures.